src/main/java/ru/sbtqa/tag/pagefactory/PageWrapper.java

```diff
@@ -142,7 +142,7 @@ public Page changeUrlByTitle(String packageName, String title) throws PageInitia
             } else {
                 try {
                     URL currentUrl = new URL(PageFactory.getWebDriver().getCurrentUrl());
```

Comment:
Current url is fetched as a URL instance, why not build it with the same class, instead of string concatenations and COLON_TWO_SLASHES variables? Or use https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/UriBuilder.html or https://github.com/mikaelhg/urlbuilder for URL buildings